dd-sdk-android-ndk/src/main/cpp/utils/backtrace-handler.cpp

```diff
@@ -41,51 +41,76 @@ namespace {
         return _URC_NO_REASON;
     }
 
-
     size_t capture_backtrace(uintptr_t *buffer, size_t max) {
         BacktraceState state = {buffer, buffer + max};
         // unwinds the backtrace and fills the buffer with stack lines addresses
         _Unwind_Backtrace(unwind_callback, &state);
         return state.current - buffer;
     }
 
-    const char *get_line_symbol(const void *addr) {
-        const char *symbol = "";
+    std::string address_to_hexa(uintptr_t address) {
+        char address_as_hexa[20];
+        #pragma clang diagnostic push
+        #pragma clang diagnostic ignored "-Wformat"
```

Comment:
Same here.

Reply:
Ah...was just a wrong copy paste. Tks for spotting this.